test/project/createNewPythonProject.test.ts

```diff
@@ -25,16 +25,16 @@ suite('Create New Python Project', () => {
         }
         this.timeout(2 * 60 * 1000);
 
-        const alias: string = process.platform === 'win32' ? 'py -3.7' : 'python3.7';
+        const alias: string = process.platform === 'win32' ? 'py -3.9' : 'python3.9';
         await runWithTestActionContext('createProject', async context => {
-            await createAndValidateProject(context, { ...getPythonValidateOptions('.venv', FuncVersion.v2), inputs: [/enter/i, alias] });
+            await createAndValidateProject(context, { ...getPythonValidateOptions('.venv', FuncVersion.v4), inputs: [/Model V1/, /enter/i, alias] });
```

Comment:
nit: you could make `/Model V1/` a constant and use it throughout this file.

Reply:
🫡